LoopKit/TherapySettings.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  TherapySettings.swift
+//  LoopKit
+//
+//  Created by Anna Quinlan on 7/6/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+
+public struct TherapySettings: Equatable {
+    
+    public var glucoseTargetRangeSchedule: GlucoseRangeSchedule?
+
+    public var preMealTargetRange: DoubleRange?
+
+    public var legacyWorkoutTargetRange: DoubleRange?
+
+    public var maximumBasalRatePerHour: Double?
+
+    public var maximumBolus: Double?
+
+    public var suspendThreshold: GlucoseThreshold?
+    
+    public var insulinSensitivitySchedule: InsulinSensitivitySchedule?
+    
+    public var carbRatioSchedule: CarbRatioSchedule?
+    
+    public init(
+        glucoseTargetRangeSchedule: GlucoseRangeSchedule? = nil,
+        preMealTargetRange: DoubleRange? = nil,
+        legacyWorkoutTargetRange: DoubleRange? = nil,
+        maximumBasalRatePerHour: Double? = nil,
+        maximumBolus: Double? = nil,
+        suspendThreshold: GlucoseThreshold? = nil,
+        insulinSensitivitySchedule: InsulinSensitivitySchedule? = nil,
+        carbRatioSchedule: CarbRatioSchedule? = nil
+    ){
+        self.glucoseTargetRangeSchedule = glucoseTargetRangeSchedule
+        self.preMealTargetRange = preMealTargetRange
+        self.maximumBasalRatePerHour = maximumBasalRatePerHour
+        self.maximumBolus = maximumBolus
+        self.suspendThreshold = suspendThreshold
+        self.insulinSensitivitySchedule = insulinSensitivitySchedule
+        self.carbRatioSchedule = carbRatioSchedule
+    }
+}
+
+extension TherapySettings: RawRepresentable {
```

Comment:
I was thinking for storage of the object if we were in the middle of the flow, but I now realize this isn't `Codable` so that wouldn't be the case. Removed, though it would be good to talk about tracking progress during the therapy flow.

Reply:
@ps2 This was something that came up in a meeting I had with Anna; I asked her how we were planning on saving progress state through the onboarding flow if the user gets half-way and then gets interrupted (and the app gets ejected from memory).